_data/companies.yml

```diff
@@ -1222,3 +1222,10 @@
   visitors: "?"
   events: Restricted
   last_update: 2020-03-11
+  
+- name: Lovoo 
+  wfh: Required 
+  travel: Restricted 
+  visitors: Restricted 
+  events: Restricted 
+  last_update: 2020-03-13 
```

Comment:
Is that a typo or are you based on a Pacific island?

Reply:
typo, should be 2020-03-12

Reply:
No problem. I fixed it while taking care of the merge conflict.

Reply:
thx :)